src/Algebra/Graph/AdjacencyIntMap/Algorithm.hs

```diff
@@ -40,65 +40,41 @@ import Data.Tree
 
 -- | Compute the /breadth-first search/ forest of a graph, such that
 --   adjacent vertices are explored in increasing order with respect
---   to their 'Ord' instance.
--- 
---   Complexity: /O((n+m)*min(n,W))/ time and /O(n)/ space.
---
--- @
--- bfsForest 'empty'                         == []
--- 'forest' (bfsForest $ 'edge' 1 1)           == 'vertex' 1
--- 'forest' (bfsForest $ 'edge' 1 2)           == 'edge' 1 2
--- 'forest' (bfsForest $ 'edge' 2 1)           == 'vertices' [1,2]
--- 'isSubgraphOf' ('forest' $ bfsForest x) x   == True
--- bfsForest . 'forest' . bfsForest          == bfsForest
--- bfsForest (3 * (1 + 4) * (1 + 5))       == [ Node { rootLabel = 1
---                                                   , subForest = [ Node { rootLabel = 5
---                                                                        , subForest = [] }]}
---                                            , Node { rootLabel = 3
---                                                   , subForest = [ Node { rootLabel = 4
---                                                                        , subForest = [] }]}]
--- 'forest' (bfsForest ('circuit' [1..5] + 'circuit' [5,4..1])) == 'path' [1,2,3] + 'path' [1,5,4]
--- @
-bfsForest :: AdjacencyIntMap -> Forest Int
-bfsForest g = bfsForestFrom' (vertexList g) g
-
--- | Like 'bfsForest', but the traversal is seeded by a list of
---   vertices. Vertices not in the graph are ignored.
+--   to their 'Ord' instance. The search is seeded by a list of
+--   argument vertices that will be the roots of the resulting
+--   forest. Argument vertices not in the graph are ignored.
 --
 --   Let /L/ be the number of seed vertices. Complexity:
 --   /O((L+m)*min(n,W))/ time and /O(n)/ space.
 --
 -- @
--- 'forest' (bfsForestFrom [1,2] $ 'edge' 1 2)      == 'vertices' [1,2]
--- 'forest' (bfsForestFrom [2]   $ 'edge' 1 2)      == 'vertex' 2
--- 'forest' (bfsForestFrom [3]   $ 'edge' 1 2)      == 'empty'
--- 'forest' (bfsForestFrom [2,1] $ 'edge' 1 2)      == 'vertices' [1,2]
--- 'isSubgraphOf' ('forest' $ bfsForestFrom vs x) x == True
--- bfsForestFrom ('vertexList' x) x               == 'bfsForest' x
--- bfsForestFrom vs ('vertices' vs)               == 'map' (\v -> Node v []) ('nub' vs)
--- bfsForestFrom [] x                           == []
--- bfsForestFrom [1,4] (3 * (1 + 4) * (1 + 5))  == [ Node { rootLabel = 1
+-- 'forest' (bfsForest [1,2] $ 'edge' 1 2)      == 'vertices' [1,2]
+-- 'forest' (bfsForest [2]   $ 'edge' 1 2)      == 'vertex' 2
+-- 'forest' (bfsForest [3]   $ 'edge' 1 2)      == 'empty'
+-- 'forest' (bfsForest [2,1] $ 'edge' 1 2)      == 'vertices' [1,2]
+-- 'isSubgraphOf' ('forest' $ bfsForest vs x) x == True
+-- bfsForest vs ('vertices' vs)               == 'map' (\v -> Node v []) ('nub' vs)
+-- bfsForest [] x                           == []
+-- bfsForest [1,4] (3 * (1 + 4) * (1 + 5))  == [ Node { rootLabel = 1
 --                                                        , subForest = [ Node { rootLabel = 5
```

Comment:
I suspect the alignment is lost here.

Reply:
It actually seems to be ok. It's still unaligned with
```
-- 'forest' (bfsForest [3] ('circuit' [1..5] + 'circuit' [5,4..1])) == 'path' [3,2,1] + 'path' [3,4,5]
```
though. I think it makes sense to push the others out to match?

Reply:
One issue is that the example before it has quite long lines with the search tree spelled out.

Reply:
I meant the alignment between `{` and `,` in the subsequent line :) It's a nit pick, don't worry about it!

Reply:
Ah I see! I'll get it in the next wave. I think there's still one or two things out of sync anyway.

There are a lot of docs/tests where the text uses `$` but the tests use the necessary parens. I was wondering if it would be better if they matched more closely? It would make it easier to update I think

Reply:
> There are a lot of docs/tests where the text uses `$` but the tests use the necessary parens. I was wondering if it would be better if they matched more closely? It would make it easier to update I think

Yeah, there are a few such cases. I guess it would be better to switch to using parentheses instead of providing examples with `$` that don't actually compile, so feel free to fix this.